packages/crowdstrike/data_stream/fdr/manifest.yml

```
@@ -236,10 +236,19 @@ streams:
       - name: max_number_of_messages
         type: integer
         title: Maximum Concurrent SQS Messages
-        description: The maximum number of SQS messages that can be in flight at any time.
+        description: Deprecated in agent version 8.16.0, this parameter is ignored if present, use number_of_workers instead. The maximum number of SQS messages that can be inflight at any time.
```

```suggestion
        description: Deprecated in agent version 8.16.0, this parameter is ignored if present, use `Number of Workers` instead. The maximum number of SQS messages that can be inflight at any time.
```
As users see `title` instead of `name` for these options.

---

@kcreddy thanks! I saw this mentioned in other issue and updated it as well.